building.rb

```diff
@@ -1,2 +1,30 @@
 class Building
+
+  attr_accessor :apartments
+
+  def initialize(args={})
+    @address          = args.fetch(:address, '')
+    @apartments       = args.fetch(:apartments, [])
+  end
+
+  def total_room_count
+    @apartments.inject(0) { |sum, apartment| sum += apartment.rooms.length }
+  end
```

Comment:
Good, but also you can use here method `room_count` which you defined in class  `Apartment`:

```
 def total_room_count
   partments.inject(0) { |sum, apartment| sum += apartment.room_count }
 end
```